packages/terra-form/lib/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$terra-form-required-color: $terra-red-70;
+$terra-form-required-indicator: '*';
+$terra-form-error-color: $terra-red-70;
+$terra-form-label-font-weight: bold;
+$terra-form-legend-font-weight: bold;
+$terra-form-field-border: 0;
+$terra-form-legend-color: $terra-dark;
+$terra-form-input-border: $terra-grey-15 solid 1px;
```

Comment:
@neilpfeiffer Did we decided if themeable variables like border is themeable by color or also by border width, etc?